BLL/DTO/CandidateDTO.cs

```diff
@@ -18,6 +18,7 @@ public class CandidateDTO
         public string Skype { get; set; }
         public int SpecializationID { get; set; }
         public int CityID { get; set; }
+        public int CountryID { get; set; }
```

Comment:
You changed candidate dto model and candidate model but you didn't change methods in endpoints. Please, delete this changes, if you are not going to do it

Reply:
We anyway need this change :(